core/src/scene/drawRule.h

```diff
@@ -37,11 +38,10 @@ class StyleContext;
 struct DrawRuleData {
 
     std::vector<StyleParam> parameters;
-    std::string name;
-    int id;
+    std::string styleName;
+    int styleId;
```

Comment:
Thanks! I'll finish reviewing `style-builder` before I come back to this, just wanted to clear up any potential confusion here.